MAINTAINERS

```diff
@@ -1,5 +1,6 @@
 Alexey Makhov <amakhov@avito.ru> (@makhov)
 Andrey Nering <andrey.nering@gmail.com> (@andreynering)
+Avelino <thiago@avelino.xxx> (@avelino)
```

Comment:
If you want to be as a maintainer, please See [CONTRIBUTING.md](CONTRIBUTING.md) and apply on gitter.im. Or maybe you want to add yourself to CONTRIBUTIORS.